include/mbgl/map/map.hpp

```diff
@@ -147,6 +159,7 @@ class Map : private util::noncopyable {
 
     inline const TransformState &getState() const { return state; }
     inline std::chrono::steady_clock::time_point getTime() const { return animationTime; }
+    inline util::ptr<AnnotationManager> getAnnotationManager() const { return annotationManager; }
```

Comment:
That requires the member variable to move from `util::ptr<AnnotationManager>` to `AnnotationManager`, and that can't be forward declared, which then gets to the same cycle with `map.hpp` and `annotation.hpp` as in https://github.com/mapbox/mapbox-gl-native/pull/1018/files#r26625693.